### DISCUSSION TITLE
Gathering the data for heating system design is quite manual at the moment. How could AI help there?

### DISCUSSION THREAD

**Discussion by esc-data-science** [alan-turing-institute/ADViCE#96] [Key sector: Built Environment]
Gathering the data for heating system design is quite manual at the moment. How could AI help there?

**Comment by samyoung-dsci** [comment]
There are quite a lot of different potential automated data sources (smart meters, sensors) that can help with getting this data. Often AI is used in turning this raw data into useful numbers for the design - e.g. calculating the heat loss from lots of sensor readings. 
 
There's also things like using LiDAR scanners to map different rooms, or automatically processing thermal images, both of which use AI under the hood. How much those directly benefit the design process will vary though, and deploying sensors comes with its own challenges!
